packages/tests-e2e/jest.config.json

```diff
@@ -1,9 +1,9 @@
 {
 	"rootDir": "../../",
 	"preset": "jest-puppeteer",
-	"setupTestFrameworkScriptFile": "<rootDir>/test/e2e/support/setup-test-framework.js",
+	"setupTestFrameworkScriptFile": "<rootDir>/packages/tests-e2e/support/setup-test-framework.js",
```

Comment:
this path seems like it'd be problematic when test-e2e is installed as a node module. Is this something you're planning on tackling in a later PR?

Reply:
Yes, good point. The same issue applies to the root directory. I think it's easier to revisit once both #12437 and #12465 are merged. I feel like having a follow-up PR with changes which try to make it reusable outside is going to be easier. For the time being, I feel like we only need to make this package private to ensure it isn't exposed to npm as it clearly isn't fully ready. However, I want to move forward so don't have to deal with the folder structure and rebases forever.